### PR TITLE
Fix: APP-2426 - Finance Page - Tokens empty state

### PR DESCRIPTION
**Pull request by Fabricevladimir**
## Description

- Adds empty state for tokens section on finance page

Task: [APP-2426](https://aragonassociation.atlassian.net/browse/APP-2426)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [x] I have tested my code on the test network.

[APP-2426]: https://aragonassociation.atlassian.net/browse/APP-2426?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/pages/finance.tsx [hunk 1]
```diff
@@ -59,91 +60,133 @@ export const Finance: React.FC = () => {
...
-      <PageWrapper
-        customHeader={
-          <HeaderContainer>
-            <Header>
-              {!isDesktop && (
+    // tokens only empty state
+    if (tokens.length === 0) {
+      return (
+        <PageWrapper includeHeader={false}>
+          <>
```
Comment (cgero-eth) [inline src/pages/finance.tsx]: Is this fragment tag here needed or can we remove it?

**Review by cgero-eth — COMMENTED** [review]
Thanks @Fabricevladimir!

File: src/pages/finance.tsx (hunk 1 above)
Comment (Fabricevladimir) [reply src/pages/finance.tsx]: The Ethereum network comes to stop without that fragment!

Comment (cgero-eth) [reply src/pages/finance.tsx]: 😱

**Review by cgero-eth — APPROVED** [review]

**Pull request merged**